src/program/symlink.py

```diff
@@ -139,11 +138,6 @@ def get_items_to_update(self, item: Union[Movie, Show, Season, Episode]) -> List
                     items.append(episode)
         return items
 
-    def update_item_folder(self, item: Union[Movie, Episode]):
-        path = get_item_path(item)
-        if path:
-            item.set("folder", path.parent.name)

```

Comment:
I'm a bit concerned about this. I can't see anywhere it's _required_, but it seems to be assumed all over the place e.g. https://github.com/rivenmedia/riven/blob/6ad6d4ddbf01593453c12b39773c07cd028bd261/src/program/libraries/symlink.py#L105

My reason for removing it is because many torrents don't put their files in a parent folder. Locally my Riven install is still working fine and symlinking correctly - with fewer broken symlinks now - but it still can't find files in the rclone root yet.